mono/utils/mono-error-internals.h

```diff
@@ -18,11 +18,17 @@ typedef struct {
 	MonoClass *klass;
 	const char *full_message;
 	const char *full_message_with_fields;
+	const char *first_argument;
 
-	void *padding [4];
-    char message [128];
+	void *padding [3];
 } MonoErrorInternal;
 
+#define error_init(error) do {	\
+	(error)->error_code = MONO_ERROR_NONE;	\
+} while (0);
```

Comment:
this doesn't do the same thing as `mono_error_init`, as it doesn't initialize `(error)->flags = 0`